sklearn/metrics/tests/test_regression.py

```diff
@@ -32,6 +33,10 @@ def test_regression_metrics(n_samples=50):
                                            np.log(1 + y_pred)))
     assert_almost_equal(mean_absolute_error(y_true, y_pred), 1.)
     assert_almost_equal(median_absolute_error(y_true, y_pred), 1.)
+    assert_almost_equal(np.around(mean_absolute_percentage_error(y_true,
+                                                                 y_pred),
+                                  decimals=2),
+                        9007199254741000.0, decimal=-13)
```

Comment:
This is a weird test. `mean_absolute_percentage_error` is really not mean to be evaluated on problem where `y_true` can have exactly zero values. I would rather not make any assertion on the exact value you get for this pathological case because nobody would care about the actual value as long as it's a finite yet large positive number.

Maybe just do something like:

```python
# Because there is a 0 in `y_true`, it does not really make sense to use MAPE for
# this regression task. Here we just check that the score is a large positive yet
# finite value:
mape = mean_absolute_percentage_error(y_true, y_pred)
assert np.isfinite(mape)
assert mape > 1e6  # arbitrary large enough positive value

# If the two values are equal and strictly positive, the expected MAPE is 0.
mape = mean_absolute_percentage_error(y_true + 1, y_pred)
assert  mape == pytest.approx(0)
```

Reply:
done